### DISCUSSION TITLE
Dev branch

### DISCUSSION THREAD

**Discussion by KaherdinTristan** [UI-Lovelace-Minimalist/UI#98] [General]
the new cards or functions that are merged into dev; are there a list somewhere that is kept "kinda" updated for these things? 
😀

**Comment by CM000n** [comment]
As soon as the 'dev' branch is merged into the 'main' branch, there will be another release note with the most important changes. 🙂
Otherwise you can always view the latest commits and pushes in the 'dev' commit history: https://github.com/UI-Lovelace-Minimalist/UI/commits/dev
